tests/static_routes_test.py

```diff
@@ -0,0 +1,266 @@
+
+        # config route del prefix 1.2.3.4/32 nexthop 30.0.0.5
+        result = runner.invoke(config.config.commands["route"].commands["del"], \
+        ["prefix", "1.2.3.4/32", "nexthop", "30.0.0.5"], obj=obj)
+        print(result.exit_code, result.output)
+        assert not '1.2.3.4/32' in db.cfgdb.get_table('STATIC_ROUTE')
+
+    def test_vrf_static_route(self):
+        db = Db()
+        runner = CliRunner()
+        obj = {'config_db':db.cfgdb}
+
+        # config route add prefix vrf Vrf-BLUE 2.2.3.4/32 nexthop 30.0.0.6
+        result = runner.invoke(config.config.commands["route"].commands["add"], \
+        ["prefix", "vrf", "Vrf-BLUE", "2.2.3.4/32", "nexthop", "30.0.0.6"], obj=obj)
+        print(result.exit_code, result.output)
+        assert ('Vrf-BLUE', '2.2.3.4/32') in db.cfgdb.get_table('STATIC_ROUTE')
+        assert db.cfgdb.get_entry('STATIC_ROUTE', 'Vrf-BLUE|2.2.3.4/32') == {'nexthop': '30.0.0.6'}
+
+        # config route del prefix vrf Vrf-BLUE 2.2.3.4/32 nexthop 30.0.0.6
+        result = runner.invoke(config.config.commands["route"].commands["del"], \
+        ["prefix", "vrf", "Vrf-BLUE", "2.2.3.4/32", "nexthop", "30.0.0.6"], obj=obj)
+        print(result.exit_code, result.output)
+        assert not ('Vrf-BLUE', '2.2.3.4/32') in db.cfgdb.get_table('STATIC_ROUTE')
+
+    def test_dest_vrf_static_route(self):
+        db = Db()
+        runner = CliRunner()
+        obj = {'config_db':db.cfgdb}
+
+        # config route add prefix 3.2.3.4/32 nexthop vrf Vrf-RED 30.0.0.6
+        result = runner.invoke(config.config.commands["route"].commands["add"], \
+        ["prefix", "3.2.3.4/32", "nexthop", "vrf", "Vrf-RED", "30.0.0.6"], obj=obj)
+        print(result.exit_code, result.output)
+        assert ('3.2.3.4/32') in db.cfgdb.get_table('STATIC_ROUTE')
+        assert db.cfgdb.get_entry('STATIC_ROUTE', '3.2.3.4/32') == {"nexthop": "30.0.0.6", "nexthop-vrf": "Vrf-RED"}
+
+        # config route del prefix 3.2.3.4/32 nexthop vrf Vrf-RED 30.0.0.6
+        result = runner.invoke(config.config.commands["route"].commands["del"], \
+        ["prefix", "3.2.3.4/32", "nexthop", "vrf", "Vrf-RED", "30.0.0.6"], obj=obj)
+        print(result.exit_code, result.output)
+        assert not ('3.2.3.4/32') in db.cfgdb.get_table('STATIC_ROUTE')
+
+    def test_several_nexthops_static_route(self):
+        db = Db()
+        runner = CliRunner()
+        obj = {'config_db':db.cfgdb}
+
+        # config route add prefix 6.2.3.4/32 nexthop "30.0.0.6,30.0.0.7"
+        result = runner.invoke(config.config.commands["route"].commands["add"], \
+        ["prefix", "6.2.3.4/32", "nexthop", "30.0.0.6,30.0.0.7"], obj=obj)
+        print(result.exit_code, result.output)
+        assert ('6.2.3.4/32') in db.cfgdb.get_table('STATIC_ROUTE')
+        assert db.cfgdb.get_entry('STATIC_ROUTE', '6.2.3.4/32') == {"nexthop": "30.0.0.6,30.0.0.7"}
+
+        # config route del prefix 6.2.3.4/32 nexthop 30.0.0.7
```

Comment:
please have a test to del prefix without any nexthops and ensure its removed from DB.